hangman.js

```diff
@@ -1,56 +1,39 @@
-const Discord = require('discord.js');
+const { Embed } = require('@discordjs/builders');
 const randomWord = require('random-word');
+const randomWordFR = require('random-word-fr');
 
 class hangman {
-    constructor(word = null, channel, players, messages, displayWordOnGameOver) {
-        this.word = word || randomWord();
+    constructor(word = null, interaction, players, messages, displayWordOnGameOver, lang) {
+        this.lang = lang || 'en';
+        this.word = word || this.lang === 'en' ? randomWord() : randomWordFR();
         this.lives = 6;
         this.progress = hangman.hyphenString(this.word.length);
         this.remaining = this.word.length;
         this.misses = [];
         this.status = 'in progress';
         this.gameOver = false;
-        this.channel = channel;
+        this.interaction = interaction;
         this.message = null;
         this.players = players;
         this.messages = messages;
         this.displayWordOnGameOver = displayWordOnGameOver;
     }
 
-    static hyphenString(n) {
-        return '-'.repeat(n);
-    }
-
-    replaceChar(char) {
-        for (let i = 0; i < this.word.length; ++i) {
-            if (this.word[i] === char) {
-                this.progress = this.progress.substring(0, i) + this.word[i] + this.progress.substring(i + this.word[i].length);
-                this.remaining--;
-            }
-        }
-    }
+    static hyphenString(n) { return '-'.repeat(n); };
+    replaceChar(char) { for (let i = 0; i < this.word.length; ++i) { if (this.word[i] === char) { this.progress = this.progress.substring(0, i) + this.word[i] + this.progress.substring(i + this.word[i].length); this.remaining--; }; }; };
```

Comment:
The code format used in the module allows to be readable and understandable by everyone. It is important to respect the writing style of the author of the project you are contributing to ;)
Some line breaks would not be too much haha

Example :
```js
myFunction(parameters) {
    if(condition) {
        // codeblock
    }
    else // Single line of code
}
// Don't forget the ";" !
```

For helping, here is an eslintrc.json file for working with eslint (maybe not up-to-date haha)
```json
{
	"extends": "eslint:recommended",
	"env": {
		"node": true,
		"es6": true
	},
	"parserOptions": {
		"ecmaVersion": 2019
	},
	"rules": {
		"brace-style": ["error", "stroustrup", { "allowSingleLine": true }],
		"comma-dangle": ["error", "always-multiline"],
		"comma-spacing": "error",
		"comma-style": "error",
		"dot-location": ["error", "property"],
		"handle-callback-err": "off",
		"indent": ["error", "tab"],
		"max-nested-callbacks": ["error", { "max": 4 }],
		"max-statements-per-line": ["error", { "max": 2 }],
		"no-console": "off",
		"no-floating-decimal": "error",
		"no-multi-spaces": "error",
		"no-multiple-empty-lines": ["error", { "max": 2, "maxEOF": 1, "maxBOF": 0 }],
		"no-shadow": ["error", { "allow": ["err", "resolve", "reject"] }],
		"no-trailing-spaces": ["error"],
		"no-var": "error",
		"object-curly-spacing": ["error", "always"],
		"prefer-const": "error",
		"quotes": ["error", "single"],
		"semi": ["error", "always"],
		"space-before-blocks": "error",
		"space-before-function-paren": ["error", {
			"anonymous": "never",
			"named": "never",
			"asyncArrow": "always"
		}],
		"space-in-parens": "error",
		"space-infix-ops": "error",
		"space-unary-ops": "error",
		"spaced-comment": "error",
		"yoda": "error"
	}
}
```